source/github-helpers/api.ts

```diff
@@ -49,7 +49,7 @@ interface RestResponse extends AnyObject {
 	ok: boolean;
 }
 
-export const escapeKey = (value: string | number): string => '_' + String(value).replace(/[ ./-]/g, '_');
+export const escapeKey = (value: string | number): string => '_' + String(value).replace(/[ ./-@]/g, '_');
```

Comment:
We should replace this with `source/helpers/hash-string.ts` instead. No point in risking this again, the key is for internal use only

Reply:
Will do

Reply:
Having second thoughts. Would this make debugging harder? Not sure why it would, but this gives a random ID instead of a readable name.

Reply:
But its possible to hit duplicates, we needed a way to garentee no duplicate names.

Reply:
> But its possible to hit duplicates

That's not wrong, but highly unlikely. These two already produce the same key, but the chances of that happening in the real world are so low:

- refined/github-repo
- refined-github/repo

A hash-based approach would avoid that too.